src/v2/guide/syntax.md

```diff
@@ -66,6 +66,8 @@ So far we've only been binding to simple property keys in our templates. But Vue
 {{ message.split('').reverse().join('') }}
 
 <div v-bind:id="'list-' + id"></div>
+
+<div v-bind:style="`background-image: url('${imageUrl}')`"></div>
```

Comment:
Thank you for your feedback!

However, It is possible only when we are using `.vue` (vue-loader).
https://vue-loader.vuejs.org/en/features/es2015.html#es2015-in-templates

I think that we should be noted necessary as a comment about it.